yamux/src/session.rs

```diff
@@ -622,6 +618,9 @@ where
                 break;
             }
 
+            self.flush(cx)?;
+            self.poll_complete(cx)?;
```

Comment:
This is a performance concern, should run a benchmark to show the difference.

Reply:
It doesn’t matter much. When the buffer is empty, only one judgment is left here.